sparse_strips/vello_common/src/lib.rs

```diff
@@ -1,9 +1,24 @@
 // Copyright 2025 the Vello Authors
 // SPDX-License-Identifier: Apache-2.0 OR MIT
 
-//! This crate contains core data structures and utilities shared across crates. It includes
-//! foundational types for path geometry, tiling, and other common operations used in both CPU and
-//! hybrid CPU/GPU rendering.
+//! This crate includes common geometry representations, tiling logic, and other fundamental components used by both Vello CPU and Vello Hybrid.
+//!
+//! ## Usage
+//!
+//! This crate should not be used on its own, and you should instead use one of the renderers which use it.
+//! At the moment, only [Vello CPU](crates.io/crates/vello_cpu) is published, and you probably want to use that.
+//!
+//! We also develop [Vello](crates.io/crates/vello), which makes use of the GPU for 2d rendering and has higher performance than Vello CPU.
+//! Vello CPU is being developed as part of work to address shortcomings in Vello.
+//! Vello does not use this crate.
+//!
+//! ## Features
+//!
+//! - Shared data structures for paths, tiles, and strips
+//! - Geometry processing utilities
+//! - Common logic for rendering stages
+//!
+//! This crate acts as a foundation for `vello_cpu` and `vello_hybrid`, providing essential components to minimize duplication.
```

Comment:
(Note, if you do not take this suggestion, make sure to fix the crates.io link to `vello_cpu` by prefixing `https://`.)

```suggestion
//! This crate acts as a foundation for `vello_cpu` and `vello_hybrid`, providing essential components to minimize duplication.
//!
//! [vello_cpu]: https://crates.io/crates/vello
```